### DISCUSSION TITLE
Next release planned for weekend of Feb. 13

### DISCUSSION THREAD

**Discussion by titinko** [titinko/utsu#75] [General]
Hi all,

Finished up all the new features planned for the upcoming release (0.4.4) so I'll spend this week on minor features/bug fixes and aim for a release next weekend.

If you have translations or features that you want to add to UTSU, please submit them before next weekend if you want them included in the release!  Thanks again to everyone who has contributed.

**Comment by iWeird** [comment]
Hello,

I’m interested in translating utsu to my own language, dutch. Note that I don’t have a lot of experience with programming, but I can translate. Where can I submit my translations? Thanks!

-iWeird

**Reply by titinko** [reply]
Instructions for creating a new translation are here: https://github.com/titinko/utsu/wiki/Adding-a-new-translation
